app/services/appropriate_bodies/claim_an_ect/register_ect.rb

```diff
@@ -19,9 +19,7 @@ def register(pending_induction_submission_params)
         # end
         ActiveRecord::Base.transaction do
           steps = [
-            update_name,
-            update_qts_awarded_on,
-            update_itt_provider_name,
+            update_teacher,
```

Comment:
@peteryates was advocating for a more granular approach, if I remember correctly, with the intent that it could be reused elsewhere. Initial ideas proposed a chaining approach (if we had update methods return `self`).

Reply:
I agree, I think it makes sense to have a single input for the params that just knows what to do with them. That's just as easy to reuse IMHO. As you say, that's the bulk upload scenario.